oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/parser/listener/MultiScopesAnalysisListener.java

```diff
@@ -392,11 +392,25 @@ private Layer fromSpanLayerValue(SpanLayer spanLayer) {
                 return Layer.CACHE;
             case UNRECOGNIZED:
                 return Layer.UNDEFINED;
+            case FAAS:
+                return Layer.FAAS;
             default:
                 throw new UnexpectedException("Can't transfer to the Layer. SpanLayer=" + spanLayer);
         }
     }
 
+    /**
+     * Simple logic, only judge ${@link Layer#FAAS} and ${@link Layer#GENERAL}.
+     */
+    private Layer fromSpanLayer(SpanLayer spanLayer) {
```

Comment:
```suggestion
    /**
     * Identify the layer of span service/instance owner. Such as  ${@link Layer#FAAS} and ${@link Layer#GENERAL}.
     */
    private Layer identifyServiceLayer(SpanLayer spanLayer) {
```

Please rename `fromSpanLayerValue` to `identifyRemoteServiceLayer`.
I think this could make the scope of 2 methods more clear. WDYT?

Reply:
yes